### PR TITLE
tm-persons, handle perdet param

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
😺
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2541)
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/787)

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -424,12 +424,15 @@ var appRouter = function (app) {
...
+    let rpF = req.query['rp.filter'];
+    const rE = /perdet_seq_num\|EQ\|(\d+)\|/;
+    let paramPerdet = rpF.match(rE)[1];
     let persons;
     // const persons = await employees.get_v3_persons_agenda_items(req.query)
     if (_.get(req.query, '["rp.columns"]', "").indexOf('ROWCOUNT') > -1) {
       persons = [{ rowcount: 1337 }]
     } else {
-      persons = await employees.get_v3_persons_agenda_items({ "request_params.page_size": 25, "request_params.page_index": 1 })
+      persons = await employees.get_v3_persons_agenda_items({ "request_params.page_size": 25, "request_params.page_index": 1, "perdet_seq_num": paramPerdet })
```
Comment (elizabeth-jimenez) [inline src/routes.js:435]: this is one of the first template EPs we mocked out, so some of the newer mock functionality hasn't been integrated into it yet - but this should do for now

**Comment by M00staff** [comment]
🎉

**Comment by M00staff** [comment]
so, since I'm using the other endpoint - do we still need this update to the mock?

**Comment by elizabeth-jimenez** [comment]
> so, since I'm using the other endpoint - do we still need this update to the mock?

no lol - closing

**Pull request closed without merging**